web/src/beta/components/fields/common/TextInput/index.tsx

```diff
@@ -32,6 +32,14 @@ const TextInput: React.FC<Props> = ({
     setCurrentValue(value ?? "");
   }, [value]);
 
+  const handlepdateValue = useCallback(
+    (newValue: string) => {
+      if (!newValue || /^ *$/.test(newValue)) setCurrentValue("");
+      onChange?.(newValue);
+    },
+    [onChange],
+  );
```

Comment:
Not quite sure what this is doing. Why do we want to in some instances set currentValue to an empty string but then pass newValue to onChange?

Reply:
the idea here that some times the user enter spaces , so I don't want to update with this spaces instead only save empty space

Reply:
This isn't really good because it alters what the user can expect without any notice.
Also, this is on the common TextInput component, which in some cases a user might want to add white space. One example is currently we don't allow align center to the page title, so they might want to center the title with spacing. I feel like that should be allowed.

Reply:
well since it is acceptable for system , so I will just remove those changes here